File: Java programming/problems on numbericals/problem81.java
/*
81. Accept N numbers from user and return the largest number. 
	
	Input :  N :   6 
	
	Elements : 85 66 3 66 93 88  
	
	Output : 93 
	
	Author : Milind Nalawade
	
	Date : 11/07/2021
	
	Language : Java 14.0.2
*/
import java.lang.*;
import java.util.*;

class Logic
{
	public static int Output(int arr[])
	{
		int max=0;
		for(int i=0;i<arr.length;i++)
		{
			if(max<arr[i])
			{
				max=arr[i];
			}
		}
		return max;
	}
}

//main class

class problem81 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print(" How many elements to take as input : ");
		int size=sobj.nextInt();
		int arr[]=new int[size];
		System.out.println();
		System.out.print("Enter "+size+" elements :-");
		
		for(int i=0;i<size;i++)
		{
			System.out.println();
			System.out.print(" Enter element number "+i+" : ");
			arr[i]=sobj.nextInt();
		}
		int ret=Output(arr);
		System.out.println();
		System.out.println(" The maximum number among the given numbers is : "+ret );
		System.out.println();
	}
}

File: Python Programming/problems on numbericals/problem81.py

'''
81. Accept N numbers from user and return the largest number. 
	
	Input :  N :   6 
	
	Elements : 85 66 3 66 93 88  
	
	Output : 93 
	
	Author : Milind Nalawade
	
	Date : 11/07/2021
	
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def Output(arr):
	max=0
	for i in range(len(arr)):
		if max<arr[i]:
			max=arr[i]
	return max
	
#=============
#Entry point function
#=============
def main():
	
	arr=[]
	size=int(input("How many element to accept for input : "))
	print()
	print(" Enter {} elements ".format(size))
	
	for i in range(size):
		print()
		arr.append(int(input(" Enter element number {} :".format(i))))
	
	print()
	print("  {} is the  maximum number among the given numbers ".format(Output(arr)))
	print()
#=====
#starter
#=====
if __name__=="__main__":
	main()